pkg/config/options.go

```diff
@@ -31,6 +42,13 @@ func (o *Options) Validate() error {
 		return fmt.Errorf("invalid --log-level: %w", err)
 	}
 	logrus.SetLevel(level)
+
+	if o.determineChanges {
```

Comment:
Why is this in `Validate()` ?

Reply:
all of the tools already run this method, so the easiest way was just to put it here.

Reply:
I moved it to a `Complete` method.